app/partials/project-card.cjsx

```diff
@@ -23,10 +30,10 @@ module.exports = React.createClass
       'data-no-avatar': true unless @props.project.avatar
 
     <Link {...linkProps} className="project-card">
-      <svg className="project-card-space-maker" viewBox="0 0 2 1" width="100%"></svg>
+      <svg className="project-card-space-maker" viewBox="0 0 2 150px" width="100%"></svg>
       <div className="details">
         <div className="name">{@props.project.display_name}</div>
         <div className="owner">{owner?.display_name ? 'LOADING'}</div>
-        <div className="description">{@props.project.description}</div>
+        <button type="button" className="ghost-button project-card-button"><Translate content="projectsCard.button" /></button>
```

Comment:
I discussed this with @heathv and he wants there to be some kind of indication that the project card is clickable and would prefer the card is the link, but needs to think about how to design it. For now, I've made the button's tabIndex -1 so they do not get tab-focus.